packages/gateway-generator/src/cli/updateSls.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
```

Reply:
@ericanderson do we have eslint set up for the copyright header?

Reply:
Its in `monorepo/eslint-config-sane/library.cjs`. We should probably split that into its own PR through.

Reply:
yupyup no worries